pkg/hooks/ssl/ssllib_hooks.go

```diff
@@ -94,6 +104,36 @@ func (s *SslHooks) installSslHooks(bpfObjects *bpf.BpfObjects, sslLibrary *link.
 	return nil
 }
 
+func (s *SslHooks) installEnvoySslHooks(bpfObjects *bpf.BpfObjects, sslLibrary *link.Executable) error {
+	var err error
+
+	s.sslWriteProbe, err = sslLibrary.Uprobe("SSL_write", bpfObjects.BpfObjs.SslWrite, nil)
+
+	if err != nil {
+		return errors.Wrap(err, 0)
+	}
+
+	s.sslWriteRetProbe, err = sslLibrary.Uretprobe("SSL_write", bpfObjects.BpfObjs.SslRetWrite, nil)
```

Comment:
are `SSL_read_ex` and `SSL_write_ex` calls are actual for envoy?

Reply:
No, envoy doesn't call SSL_read_ex and SSL_write_ex